pytorch_lightning/accelerators/cpu.py

```diff
@@ -29,3 +33,7 @@ def setup(self, trainer: "pl.Trainer") -> None:
             raise MisconfigurationException(f"Device should be CPU, got {self.root_device} instead.")
 
         return super().setup(trainer)
+
+    def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
+        """Returns dummy implementation for now."""
```

Comment:
> for now?

Are we planning to return any stats in the future?

Reply:
@ananthsub mentioned "in the future we could use something like psutil to get per-process stats https://psutil.readthedocs.io/" so potentially, but I don't think we have concrete plans. Should I remove that from the comment?

Reply:
Yes

Reply:
its coming up now: https://github.com/PyTorchLightning/pytorch-lightning/issues/11253
fyi @EricWiener